packages/graphiql/package.json

```diff
@@ -36,7 +35,7 @@
     "build-demo": "cp resources/index.html .",
     "check": "flow check",
     "dev": "babel-node test/server.js",
-    "e2e": "yarn e2e:server && cypress run && kill -9 `cat test/pid`",
+    "e2e": "yarn e2e:server && cypress run",
```

Comment:
it was causing weird issues in CI, where the last of the two would kill the first one's tests mid-run.
i also thought it would be nice to relieve the e2e server from tracking the PID using the filesystem

Reply:
:man_shrugging: Whatever. Just raised it in case it was a mistake.

Reply:
I'm pretty torn about it TBH. And there's probably a better solution that I'm not picking up on

Reply:
Meh.